api/swagger.yaml

```diff
@@ -19,18 +19,18 @@ paths:
               schema:
                 $ref: '#/components/schemas/chat_list_response'
               example:
-                - chat_id: 543
-                  user_ids: [123, 234]
-                  last_activity: "2022-10-10T18:29:48.843Z"
+                - chatId: 543
+                  dancerIds: [123, 234]
+                  lastActivity: "2022-10-10T18:29:48.843Z"
                   type: "group"
                   messages: []
-                  last_message:
+                  lastMessage:
                     text: Lorem ipsum
                     author: 123
                     id: 1234
-                    read_by_users: [123]
-                    notified_users: [123]
-                    creation_timestamp: "2022-10-10T18:29:48.843Z"
+                    readByDancers: [123]
+                    notifiedUsers: [123]
```

Comment:
Since we changed the `read_bu_users` field to `readByDancers`, shouldn't we change this field from `notified_users` to `notifiedDancers`?

Reply:
I think we could also remove this property from the payload. It is not of interest for the frontend, messages for which dancers were notified are not handled differently. It is only of interest in the backend so that it can be decided when a dancer should be notified again.

Reply:
First I thought I would certainly agree with you, that this it not needed for the frontend and for this reason we can remove it from that API.
But than I realized that this API is the one for the SoR for the chat-system not the API in den BFF :-)
And the chat-dancer, will expose the information of notified messages to maybe a component that is responsible for sending out  those notification emails. Maybe we need that kind of information than.
As of now, we do not know how we will implement those notifications. Until then I would propose leaving the information here, even if we can drop it later.
(but this is not a strong feeling, we can also drop it now a add it later when we need it. @jans510 and @SteliosRusta decide what you want ;-) )

@kai-moritz when we have kafka stable here, I would guess that would be an use case for it ;-)

Reply:
@SteliosRusta and @jans510 
What should we do: remove notified user and add it afterwards, wenn we need it? Or leave it in place an remove it when we later do not need it ;-)

Reply:
I would say as discussed remove it for now so that we can add it when it is needed and do not need to remove it when it is note needed what would be a breaking API change

Reply:
I have removed it, the code snippet seen above is already outdated ;-) please recheck